main.go

```diff
@@ -342,3 +347,24 @@ func getUserClaims(i kratos_client.Identity, cr hydra_client.OAuth2ConsentReques
 
 	return ret
 }
+
+func setUpPrometheus() *prometheus.MetricsManager {
+	mm := prometheus.NewMetricsManagerWithPrefix("identity-platform-login-ui-operator", "http", "", "", "")
+	mm.RegisterRoutes(
+		"/api/kratos/self-service/login/browser",
+		"/api/kratos/self-service/login/flows",
+		"/api/kratos/self-service/login",
+		"/api/kratos/self-service/errors",
+		"/api/consent",
+		"/consent.html",
+		"/error.html",
+		"/index.html",
+		"/login.html",
+		"/",
+		"",
+		"/oidc_error",
+		"/registration",
```

Comment:
I don't really like having to manually add each path to this list, it seems error prone.

For the server paths, couldn't we use the middleware to register them?
For the UI paths it is not that simple (we could do it on runtime, but that is not ideal). Perhaps we could scan the UI dir on startup and construct this list?

Reply:
For your second point. Sorry but what UI dir? I was under the impression that we embed everything into a single binary when deploying.